src/builder/specification.md

```diff
@@ -0,0 +1,313 @@
+# Builder API
+
+This document specifies the Builder API methods that the Consensus Layer uses to interact with external block builders.
+
+## Structures
+
+### `ExecutionPayloadV1`
+
+Mirror of [`ExecutionPayloadV1`][execution-payload].
+
+### `ExecutionPayloadHeaderV1`
+Equivalent to `ExecutionPayloadV1`, except `transactions` is replaced with `transactionsRoot`.
+- `parentHash`: `DATA`, 32 Bytes
+- `feeRecipient`:  `DATA`, 20 Bytes
+- `stateRoot`: `DATA`, 32 Bytes
+- `receiptsRoot`: `DATA`, 32 Bytes
+- `logsBloom`: `DATA`, 256 Bytes
+- `prevRandao`: `DATA`, 32 Bytes
+- `blockNumber`: `QUANTITY`, 64 Bits
+- `gasLimit`: `QUANTITY`, 64 Bits
+- `gasUsed`: `QUANTITY`, 64 Bits
+- `timestamp`: `QUANTITY`, 64 Bits
+- `extraData`: `DATA`, 0 to 32 Bytes
+- `baseFeePerGas`: `QUANTITY`, 256 Bits
+- `blockHash`: `DATA`, 32 Bytes
+- `transactionsRoot`: `DATA`, 32 Bytes
+
+### `BlindBeaconBlockV1`
```

Comment:
The beacon API uses 'blinded' rather than 'blind'.  Would be good to have consistency here.

Reply:
done